Nio/Conversations/Event Views/MessageView/MediaEventView.swift

```diff
@@ -63,27 +61,17 @@ struct MediaEventView: View {
     }
 
     var body: some View {
-        VStack(alignment: .leading, spacing: 5) {
-            senderView
-            HStack {
-                if isMe {
-                    Spacer()
-                }
-                ZStack(alignment: isMe ? .bottomTrailing : .bottomLeading) {
-                    WebImage(url: urls.first!, isAnimating: .constant(true))
-                        .resizable()
-                        .indicator(.activity)
-                        .scaledToFit()
-                        .frame(width: UIScreen.main.bounds.size.width - 100)
-                        .mask(RoundedRectangle(cornerRadius: 15))
-                    timestampView
-                        .padding(6)
-                }
-                if !isMe {
-                    Spacer()
-                }
-            }
+        VStack(alignment: self.isMe ? .trailing : .leading, spacing: 5) {
+            self.senderView
+            WebImage(url: self.urls.first!, isAnimating: .constant(true))
+                .resizable()
+                .indicator(.activity)
+                .scaledToFit()
+                .mask(RoundedRectangle(cornerRadius: 15))
```

Comment:
I realize this is disgusting, but any idea why this works on iPad, but doesn't on iPhone?

```suggestion
            HStack {
                if isMe {
                    Spacer()
                }
                WebImage(url: self.urls.first!, isAnimating: .constant(true))
                    .resizable()
                    .indicator(.activity)
                    .scaledToFit()
                    .mask(RoundedRectangle(cornerRadius: 15))
                if !isMe {
                    Spacer()
                }
            }
```

Reply:
I'd assume it's due to the sidebar.